[2023-03-29 19:20:13,739] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.start_task scheduled__2023-03-29T10:05:00+00:00 [queued]>
[2023-03-29 19:20:13,754] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.start_task scheduled__2023-03-29T10:05:00+00:00 [queued]>
[2023-03-29 19:20:13,754] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 19:20:13,755] {taskinstance.py:1357} INFO - Starting attempt 1 of 4
[2023-03-29 19:20:13,755] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 19:20:13,775] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): start_task> on 2023-03-29 10:05:00+00:00
[2023-03-29 19:20:13,785] {standard_task_runner.py:52} INFO - Started process 80 to run task
[2023-03-29 19:20:13,789] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_merge_dag', 'start_task', 'scheduled__2023-03-29T10:05:00+00:00', '--job-id', '632', '--raw', '--subdir', 'DAGS_FOLDER/data_merge.py', '--cfg-path', '/tmp/tmpg1pnlny6', '--error-file', '/tmp/tmpiet9rm38']
[2023-03-29 19:20:13,790] {standard_task_runner.py:80} INFO - Job 632: Subtask start_task
[2023-03-29 19:20:13,901] {task_command.py:369} INFO - Running <TaskInstance: data_merge_dag.start_task scheduled__2023-03-29T10:05:00+00:00 [running]> on host 0f918821725a
[2023-03-29 19:20:14,037] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=kartaca
AIRFLOW_CTX_DAG_ID=data_merge_dag
AIRFLOW_CTX_TASK_ID=start_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T10:05:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-29T10:05:00+00:00
[2023-03-29 19:20:14,039] {logging_mixin.py:115} INFO - DAG has been started...
[2023-03-29 19:20:14,039] {python.py:173} INFO - Done. Returned value was: DAG has been started...
[2023-03-29 19:20:14,089] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=data_merge_dag, task_id=start_task, execution_date=20230329T100500, start_date=20230329T192013, end_date=20230329T192014
[2023-03-29 19:20:14,163] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-29 19:20:14,236] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
